DSP_API/ThumbDV/thumbDV.c

```diff
@@ -668,10 +676,18 @@ static void _connectSerial( FT_HANDLE * ftHandle )
     do {
 
         status = FT_CreateDeviceInfoList(&numDevs);
+        if (status != FT_OK)
+        {
+            output("Unable to create Device Info \n");
```

Comment:
Do you plan to leave the output statements in the production code?

Reply:
Yes. This statement cannot be seen by the user but helpful for development